Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM silintl/php7:7.2
+MAINTAINER Mark Tompsett <mark_tompsett@sil.org>
+
+ENV REFRESHED_AT 2020-05-07
+
+# Fix timezone stuff from hanging.
+RUN apt-get update -y && echo "America/New_York" > /etc/timezone; \
+    apt-get install -y tzdata
+
+# Make sure apt has current list/updates
+# Install necessary PHP building blocks
+# Install Apache and PHP (and any needed extensions).
```

Comment:
Incorrect reference to Apache in this comment.

Reply:
Fixed.